.github/CODEOWNERS

```diff
@@ -101,3 +101,6 @@ java/code/src/com/suse/manager/webui/utils/gson/NotificationMessageJson.java @uy
 
 # Websockets
 java/code/src/com/suse/manager/webui/websocket/ @uyuni-project/frontend
+
+# Ignore translations, in the CODEOWNERS the last matching wins
+**/*.po
```

Comment:
Java has them in xml files. StringResource*.xml
And we should also exclude .pot files as well

Reply:
Done.